shotover/benches/benches/codec/cassandra.rs

```diff
@@ -30,10 +30,93 @@ fn criterion_benchmark(c: &mut Criterion) {
             },
         }))];
 
-        let (_, mut encoder) =
+        let (mut decoder, mut encoder) =
```

Comment:
I think a fresh decoder should be created for each bench.
Otherwise whether or not the encode bench is run could affect the results of the decode bench.
From a quick look at the CassandraCodecBuilder implementation I dont think any of the state that is currently shared between encoder and decoder could cause issues here, but we could very easily introduce more shared state in the future that results in the benches influencing each others results.